pkg/dns/azure/dns.go

```diff
@@ -154,3 +157,19 @@ func getARecordName(recordDomain string, zoneName string) (string, error) {
 	}
 	return strings.TrimSuffix(trimmedDomain, "."+zoneName), nil
 }
+
+// GetTagList returns a list of tags by merging the OCP default tags
+// and the user-defined tags present in the Infrastructure.Status
+func GetTagList(infraStatus *configv1.InfrastructureStatus) map[string]*string {
+	tags := map[string]*string{
+		fmt.Sprintf("kubernetes.io_cluster.%v", infraStatus.InfrastructureName): to.StringPtr("owned"),
```

Comment:
nit: Could you make the field name prefix and field value be constants?

Reply:
comment incorporated.